### PR TITLE
db_queries.php: use the same color scheme from Dashboard

### PR DESCRIPTION
**Pull request by rdwebdesign**
## What does this PR aim to accomplish?

Use a single color scheme and similar order for both pages (dashboard and db_queries):
- Blue (aqua) for `Total queries`;
- Red for `Queries blocked`;
- Yellow for `Percentage Blocked`;

_Dashboard colors:_
dashboard

_Long-term data > Query log (`db_queries.php`):_
db_queries_same_colors

## How does this PR accomplish the above?

Replacing the colors on `db_queries.php` page.

## Link documentation PRs if any are needed to support this PR

none

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

File: db_queries.php [hunk 1]
```diff
@@ -84,21 +84,21 @@
         <!-- small box -->
         <div class="small-box bg-aqua no-user-select">
             <div class="inner">
-                <h3 class="statistic" id="queries_blocked_exact">---</h3>
-                <p>Queries Blocked</p>
+                <h3 class="statistic" id="dns_queries">---</h3>
+                <p>Queries Total</p>
```
Comment (yubiuser) [inline db_queries.php]:
```suggestion
                <p>Total queries</p>
```

**Review by yubiuser — COMMENTED** [review]
Do you want to use the same wording as well?

L127 would need a change then too.

**Review by yubiuser — COMMENTED** [review]
Being a Stickler:
At the dashboard `Total queries` -> `Total Queries`

**Comment by rdwebdesign** [comment]
I don't want to fix the Dashboard here because there are other similar "mixed case" items and I don't think this is related to this PR.

If we want to follow a standard, we first need to decide where we want "Capitalize Every Word" and where we want to "Capitalize only the first letter of each sentence" and then fix it everywhere.

Currently there is no standard:

mixedcase

**Review by yubiuser — APPROVED** [review]
Keep the dashboard as it is. It's ~~a mess~~ organically grown.

**Pull request merged**